generator/internal/genclient/service_config.go

```diff
@@ -18,8 +18,10 @@ import (
 	"fmt"
 	"os"
 
+	"github.com/ghodss/yaml"
 	"google.golang.org/genproto/googleapis/api/serviceconfig"
-	"gopkg.in/yaml.v3"
+	"google.golang.org/protobuf/encoding/protojson"
+	_ "google.golang.org/protobuf/types/known/wrapperspb"
```

Comment:
What is this needed for? If it is needed for something could you add a comment.

Reply:
Not needed, I neglected to remove this after adding it during troubleshooting. Removed.